api/src/test/scala/slog4s/MockClock.scala

```diff
@@ -0,0 +1,34 @@
+package slog4s
+
+import java.util.concurrent.TimeUnit
+
+import cats.Monad
+import cats.effect.concurrent.Ref
+import cats.effect.{Clock, Sync}
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+
+import scala.concurrent.duration.Duration
+
+class MockClock[F[_]](val real: Ref[F, Long], val mono: Ref[F, Long])(
```

Comment:
Shouldn't this class be part of the `testkit`?

Reply:
It is used only for test of the testkit. I don't want to expose more public API that needed.